Runtime/Interaction/HPUIInteractorRayAngle.cs

```diff
@@ -87,6 +87,29 @@ public static void GetDirection(float x, float z, in float3 right, in float3 for
 
             direction = yDist * up + zDist * forward + xDist * right;
         }
+        
+        [BurstCompile]
```

Comment:
This won't burst compile. This should be static and not use Vector3. Also if the previous function is no longer used, replace that.